mobile/library/swift/EngineBuilder.swift

```diff
@@ -472,6 +473,18 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set a runtime guard with the provided value.
+  ///
+  /// - parameter name:  the name of the runtime guard, e.g. test_feature_false.
+  /// - parameter value: the value fo the runtime guard.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func addRuntimeGuard(name: String, value: String) -> Self {
```

Comment:
I think it's appropriate to omit the parameter labels for this function.

```suggestion
  public func addRuntimeGuard(_ name: String, _ value: Bool) -> Self {
```